### ISSUE TITLE
CommComm Restructuring Efforts Meeting 2018-05-28 (date to be changed)

### ISSUE THREAD

**Issue by chowdhurian** [nodejs/community-committee#309]
Note: This is **not** the next meeting of the title "Community Committee Membership Discussion Meeting" in the Node.js calendar. As of the date this issue is opened, there is [one sooner](https://github.com/nodejs/community-committee/issues/307). However, it had been indicated that the 28th of May is a national holiday is at least one of the regions from where regular attendees of this meeting are located.

Thanks @dshaw! 👍🏽

_(to be changed)_ meeting time: __28th of May 2018, 6:00PM UTC__

Previous minutes: TBA
Current minutes: TBA

## Invited

* @nodejs/community-committee 

## Joining the meeting

* link for participants: Will be added a few minutes before meeting starts
* For those who just want to watch:
* youtube admin page: https://www.youtube.com/my_live_events?filter=scheduled

**Comment by joesepi** [comment]
If its today, I can't make it. :(

**Comment by Tiriel** [comment]
I don't think there's a new date yet. @chowdhurian do you want to set a date/create a doodle or should I do it?

**Comment by chowdhurian** [comment]
@Tiriel The new date is now June 25th 2018. This meeting slot, previously tackling CommComm Membership, is now being used towards addressing Individual Membership (#337)

The language generated by several rounds of this Restructuring Effort has made it into several PRs 🎉 🎉 🎉

We can continue the discussion async using issues. Another meeting can be convened when we need it.

* emeriti [#302](https://github.com/nodejs/community-committee/issues/302)
* Defining working groups, and empowering other groups
* How do we track big changes happening from a sub committee?

The full minutes around the above from the last meeting #307: https://docs.google.com/document/d/1wy-53bWsoFeZWLDQlyS_5REsoMkWa6CzXxeZ55QrDNE/edit

**Comment by Tiriel** [comment]
I personally think that the work we have left now can be done, or at least started async, with provisions on one question:

> Defining working groups, and empowering other groups

Following some parts of the discussion in the Governance talk at the end of the Collab Summit, I think we should start by asking ourselves "Do we _**need**_ Working Groups in the CommComm?".
We started planning on getting WGs to the CommComm because that's how it's done in the TSC, but I remember that during said discussion, we came to the conclusion that we don't necessarily *have* to mirror this structure in the CommComm, because we don't really have the need for chartered WGs.

So all in all, I think we should start by debating the need before the how.
And to be fair, I think we should start by tackling the other points, they can be dealt with asynchronously more easily than this one, so it can be saved for later when we can program meetings again.

Any thoughts?
